azurerm/internal/services/network/firewall_resource.go

```diff
@@ -51,9 +51,39 @@ func resourceArmFirewall() *schema.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupName(),
 
+			// TODO 3.0: change this to required
+			"sku_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(network.AZFWHub),
+					string(network.AZFWVNet),
+				}, false),
+			},
+
+			// TODO 3.0: change this to required
```

Comment:
Does this default to a value right now? if so can we just defaut to it?

Reply:
Yes, currently the default sku name is: `Standard`.

But same question as above.